test/Parse/line-directive.swift

```diff
@@ -53,3 +53,9 @@ enum E {
     case C, D
 #sourceLocation()
 }
+
+let a = [
+// expected-untestable-error@+2 {{expected member name following '.'}}
```

Comment:
You can check this by piping text diagnostics to FileCheck instead of using `-verify` (with a separate RUN line, I guess).

Reply:
I’ve pushed an amend commit with your suggestion.